<launch>
	<node pkg="mapconversion" type="map_conversion_oct_node" name="map_conversion" respawn="false" output="screen"> 
	    <remap from="octomap" to="octomap_binary" />
	    <param name="minimum_z" value="1" />
	    <param name="minimum_occupancy" value="10" />
	    <param name="map_frame" value="map"/>
	    <param name="max_slope_ugv" value="0.2" />
	    <param name="slope_estimation_size" value="2" />
	</node>
</launch>
